javascripts/main.js

```diff
@@ -5,15 +5,19 @@ var tree = {};
 
 treeButton.addEventListener("click", function(){ createTree(tree);});
 
-
+window.addEventListener('keyup', function (e) {
```

Comment:
The event listener should be on the input elements, not the whole window. Although this does work, if you have another listener for a keyup event it may not work like you want it to.